script.js

```diff
@@ -1,50 +1,52 @@
 /* global MutationObserver */
 
-function updateStrategy () {
+function updateStrategy() {
   const branch = document.querySelector('.head-ref').textContent;
```

Comment:
Don't we want this to base .base-ref? .head-ref is the branch name being merged.

Reply:
let's keep it as is, we had this setting previously with no issues